src/utils/preparePrompt.ts

```diff
@@ -18,11 +25,17 @@ export const prepareProjectPrompt = (projectRoot: vscode.WorkspaceFolder, userna
 
     const promptImage = vscode.workspace.getConfiguration('docker.make-runbook').get('prompt-image') as string;
 
-    const result = spawnSync('docker', ['run', '--rm', "-v", "/var/run/docker.sock:/var/run/docker.sock", "--mount", "type=volume,source=docker-prompts,target=/prompts", promptImage, projectRoot.uri.fsPath, username, platform, promptType]);
+    const promptImageArgs = ['run', '--rm', "-v", "/var/run/docker.sock:/var/run/docker.sock", "--mount", "type=volume,source=docker-prompts,target=/prompts", promptImage, projectRoot.uri.fsPath, username, platform, promptType];
+
+    const result = spawnSync('docker', promptImageArgs);
 
     if (result.error) {
         throw result.error;
     }
-
-    return JSON.parse(result.stdout.toString());
+    try {
+        return JSON.parse(result.stdout.toString());
+    }
+    catch (e) {
+        throw new Error(`Expected JSON from ${promptImage}, got STDOUT: ${result.stdout.toString()} STDERR: ${result.stderr.toString()} ERR: ${(result.error || "N/A").toString()}`);
```

Comment:
If there's a zero exit code and a non json payload then this is an internal error.  But we need users to see the stderr for non-zero exit codes because they may have passed invalid github refs, or the content at that github ref might have issues, and that's a user facing problem.

Reply:
> But we need users to see the stderr for non-zero exit codes because they may have passed invalid github refs, or the content at that github ref might have issues, and that's a user facing problem

They will still see stderr here. I was using it extensively during testing prompts.

Reply:
okay, cool